src/utils.js

```diff
@@ -154,3 +154,16 @@ export function mapToCssModules(className, cssModule) {
   if (!cssModule) return className;
   return className.split(' ').map(c => cssModule[c] || c).join(' ');
 }
+
+/**
+ * Returns a new object with the key/value pairs from `obj` that are not in the array `omitKeys`.
+ */
+export function omit(obj, omitKeys) {
+  const result = {};
+  Object.keys(obj).forEach(key => {
```

Comment:
Right now we have the following 2 ways that omit is used:

```js
omit(this.props, ['some', 'keys', 'to', 'omit'])

omit(this.props, Object.keys(propTypes))
```

That means that both the object and the keys to omit are always passed in.

Right now if you pass in `null` for either parameter, it will blow up because it will execute `Object.keys(null)` or `null.indexOf(key)`.

Would you expect `obj` or `omitKeys` to be able to be `null` ?